server/routes/utils/adHelpers.ts

```diff
@@ -129,25 +129,21 @@ export const anomalyResultMapper = (anomalyResults: any[]): AnomalyResults => {
       ...others,
       anomalyGrade:
         rest.anomalyGrade != null && rest.anomalyGrade > 0
-          ? Number.parseFloat(rest.anomalyGrade).toFixed(3)
+          ? Number.parseFloat(rest.anomalyGrade).toFixed(2)
           : 0,
       confidence:
         rest.anomalyGrade != null && rest.anomalyGrade > 0
-          ? Number.parseFloat(rest.confidence).toFixed(3)
+          ? Number.parseFloat(rest.confidence).toFixed(2)
           : 0,
       startTime: rest.dataStartTime,
       endTime: rest.dataEndTime,
-      plotTime:
-        rest.dataStartTime +
-        Math.floor((rest.dataEndTime - rest.dataStartTime) / 2),
+      plotTime: rest.dataEndTime,
```

Comment:
just want to confirm, should we use data end time or data start time as plot time? Dashboard Live chart uses data start time. I guess we may keep consistency.

Reply:
How about we use data end time? If we use start time, hard to compare alerts history and anomalies on AD result chart.